pkg/sync/cluster.go

```diff
@@ -91,13 +94,24 @@ func httpRequest(url string, body []byte) (ans []byte, err error) {
 var sendStatMu sync.Mutex
 
 func sendStats(addr string) {
+	var dirs []string
+LOOP:
+	for len(dirs) < 5000 {
```

Comment:
How to retry once it reach the limit?

Reply:
Send no more than 5000 each time, if it exceeds 5000, leave it for the next send, and before the worker exits, check whether all have been sent.

Reply:
is it neccessary?